concept.tex

```diff
@@ -151,7 +156,7 @@ \subsubsection{Reproducibility}\label{sec:concept}\label{sec:reproducibility}
 
 Such tables and figures in a publication may be computed from the analysis of
 some type of raw data which could originate from an experiment, another publication,
-a data base, post-processing of another dataset or from executing computer
+a data base, post-processing of another data set or from executing computer
```

Comment:
Single-word dataset is more common and preferred to 'data set' nowadays, I think.